### PR TITLE
DOM Assignments 

### PR DESCRIPTION
**Pull request by rishabhtatia1**

### CODE REVIEW

File: Evenoddpartition.js [hunk 1]
```diff
@@ -7,17 +7,17 @@ evenOddPartition([5, 8, 9, 2, 0]) ➞ [[8, 2, 0], [5, 9]]
...
-			}
-	}
-	let all = [evenarr ,oddarr];
-	return all;
+const evenOddPartition = (arr) => {
+    let evenArr = [],
+        oddArr = [],
+        all = [];
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] % 2 === 0) {
```
Comment (visha12l) [inline Evenoddpartition.js]: use ternary opearator

File: JS DOM Basic Assignments/Changecolor/script.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const colorBtn = document.querySelector('.colorBtn');
+const bodyBkg = document.querySelector('body');
+
```
Comment (visha12l) [inline JS DOM Basic Assignments/Changecolor/script.js]: no blank line

File: JS DOM Basic Assignments/Counter/script.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+let counter = document.querySelector('.counter');
+const addCount = document.querySelector('#addCntBtn');
+const decCount = document.querySelector('#lowerCntBtn');
+let count = 0;
+addCount.addEventListener('click', incrementCount);
+decCount.addEventListener('click', decrementCount);
+
+function incrementCount() {
```
Comment (visha12l) [inline JS DOM Basic Assignments/Counter/script.js]: convert this to es6 function

File: JS DOM Basic Assignments/Counter/script.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+let counter = document.querySelector('.counter');
+const addCount = document.querySelector('#addCntBtn');
```
Comment (visha12l) [inline JS DOM Basic Assignments/Counter/script.js]: use only one const and let per scope

File: JS DOM Basic Assignments/Counter/script.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+const decCount = document.querySelector('#lowerCntBtn');
+let count = 0;
+addCount.addEventListener('click', incrementCount);
+decCount.addEventListener('click', decrementCount);
+
+function incrementCount() {
+    count++;
+    counter.innerHTML = count;
+    if (counter.innerHTML > '0') {
```
Comment (visha12l) [inline JS DOM Basic Assignments/Counter/script.js]: ternary operator

File: JS DOM Basic Assignments/Counter/script.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    }], {
+        duration: 500,
+        fill: 'forwards'
+    });
+}
+
+function decrementCount() {
+    count--;
+    counter.innerHTML = count;
+    if (counter.innerHTML < '0') {
```
Comment (visha12l) [inline JS DOM Basic Assignments/Counter/script.js]: ternary operator

File: JS DOM Basic Assignments/Hexcolor/script.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const hexNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 'A', 'B', 'C', 'D', 'E', 'F'];
+const hexBtn = document.querySelector('.hexBtn');
```
Comment (visha12l) [inline JS DOM Basic Assignments/Hexcolor/script.js]: only one const

File: JS DOM Basic Assignments/Hexcolor/script.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+const bodyBck = document.querySelector('body');
+const hex = document.querySelector('.hex');
+
+hexBtn.addEventListener('click', getHex);
+
+function getHex() {
```
Comment (visha12l) [inline JS DOM Basic Assignments/Hexcolor/script.js]: es6 function

File: JS DOM Basic Assignments/Message/script.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+const sendBtn = document.querySelector('#sendBtn');
+const messageIn = document.querySelector('#messageIn');
```
Comment (visha12l) [inline JS DOM Basic Assignments/Message/script.js]: one const per declaration

File: JS DOM Basic Assignments/Quotes/script.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+        name: "Tony Robbins",
+        quote: "If you do what you\'ve always done, you\'ll get what you\'ve always gotten."
+    },
+    {
+        name: "Aristotle",
+        quote: "There is only one way to avoid criticism: do nothing, say nothing, and be nothing."
+    }
+];
+const quoteBtn = document.querySelector('#quoteBtn');
+const quote = document.querySelector('#quote');
```
Comment (visha12l) [inline JS DOM Basic Assignments/Quotes/script.js]: one const per declaration

File: JS DOM Basic Assignments/Quotes/script.js [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+const quoteAuthor = document.querySelector('#quoteAuthor');
+
+quoteBtn.addEventListener('click', displayQuote);
+
+function displayQuote() {
```
Comment (visha12l) [inline JS DOM Basic Assignments/Quotes/script.js]: arrow function

File: productdivisiblebysum.js [hunk 12]
```diff
@@ -7,30 +7,25 @@ divisible[3, 2, 4, 2] ➞ false
...
-			return true;
-			else
-				return false;
-		}
-
+const divisible = (arr) => {
+    let sum = 0,
+      product = 1,
+      i;
+    for (i = 0; i < arr.length; i++) {
```
Comment (visha12l) [inline productdivisiblebysum.js]: both can be calculated in one loop

File: JS DOM Basic Assignments/Counter/script.js (hunk 3 above)
Comment (rishabhtatia1) [reply JS DOM Basic Assignments/Counter/script.js]: So will need to declare const function before call
Correct ?

**Pull request merged**